Overview.src.html

```diff
@@ -3842,7 +3842,8 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 <h3>Body mixin</h3>
 
 <pre class=idl>typedef any <dfn>JSON</dfn>;
-typedef (<span data-anolis-spec=fileapi>Blob</span> or BufferSource or <span data-anolis-spec=xhr>FormData</span> or <span data-anolis-spec=url>URLSearchParams</span> or USVString) <dfn>BodyInit</dfn>;</pre>
+typedef (<span data-anolis-spec=fileapi>Blob</span> or BufferSource or <span data-anolis-spec=xhr>FormData</span> or <span data-anolis-spec=url>URLSearchParams</span> or USVString) <dfn>RequestBodyInit</dfn>;
+typedef (<span data-anolis-spec=fileapi>Blob</span> or BufferSource or <span data-anolis-spec=xhr>FormData</span> or <span data-anolis-spec=url>URLSearchParams</span> or USVString or <code title=concept-ReadableStream>ReadableStream</code>) <dfn>ResponseBodyInit</dfn>;</pre>
```

Comment:
Fixed. I'd like to define ResponseBodyInit here because with ResponseBodyInit it's easier to understand why we have ReadableStream clause in "extract a body and a Content-Type value" algorithm.